Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/GDSError/GDSErrorViewController.swift

```diff
@@ -0,0 +1,89 @@
+import UIKit
+
+/// View controller for `Error` screen
+///     - `errorImageView` (type: `String`)
+///     - `errorTitle` (type: `UILabel`)
+///     - `errorBody` (type: `UILabel`)
+///     - `primaryButton`  (type: ``RoundedButton`` inherits from ``SecondaryButton``)
+///     - `secondaryButton`  (type: ``SecondaryButton`` inherits from ``UIButton``)
+///
+///  A navigation item can be configured:
+///  If viewModel conforms to BaseViewModel:
+///  - A back button can be set via the `hideBackButton` boolean property on the view controller
+///  - A right bar button can be set via the`rightBarButtonTitle` string property on the view controller
+///  - A `viewWillAppear` lifecycle event triggers the `didAppear` method in the viewModel.
+///  - A `dismissScreen` lifecycle event triggers the `didDismiss` method in the viewModel.
+///
+/// The `primaryButton`'s action is set from the ``ButtonViewModel`` in the viewModel.
+/// The `secondaryButton`'s action is set from the ``ButtonViewModel`` in the viewModel.
+public final class GDSErrorViewController: BaseViewController {
+    public override var nibName: String? { "GDSError" }
+    
+    private let viewModel: GDSErrorViewModel
+
+    public init(viewModel: GDSErrorViewModel) {
+        self.viewModel = viewModel
+        super.init(viewModel: viewModel as? BaseViewModel, nibName: "Error", bundle: .module)
+    }
+    
+    @available(*, unavailable, renamed: "init(coordinator:)")
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    @IBOutlet private var errorImageView: UIImageView! {
+        didSet {
+            let font = UIFont(style: .largeTitle, weight: .light)
+            let configuration = UIImage.SymbolConfiguration(font: font, scale: .large)
+            errorImageView.image = UIImage(systemName: viewModel.image, withConfiguration: configuration)
+            errorImageView.accessibilityIdentifier = "error-image"
+        }
+    }
+    
+    @IBOutlet private var errorTitle: UILabel! {
```

Comment:
On almost (perhaps every?) other screen we have called the title label `titleLabel`. Please could we follow the same convention. 

To change its name, I would suggest using the `refactor -> rename` tool because doing this manually will be quite tricky with the storyboard.

Reply:
done ✅ and I will remember this convention for next time!